spec/rails_helper.rb

```diff
@@ -11,14 +11,26 @@
 require 'webmock/rspec'
 require 'vcr'
 require 'capybara/rails'
+require 'capybara-screenshot/rspec'
 require 'database_cleaner'
 require 'active_fedora/cleaner'
 require 'selenium-webdriver'
 require 'webdrivers'
 require 'i18n/debug' if ENV['I18N_DEBUG']
 require 'byebug' unless ENV['CI']
 require 'noid/rails/rspec'
+# without lines 23-32, screenshots and html captured from failing specs are blank
```

Comment:
will this work if you do it in a support file? you include the supports next, so I bet it will and then it doesn't need to clutter up rails_helper